app/helpers/alchemy/pages_helper.rb

```diff
@@ -266,12 +266,7 @@ def render_breadcrumb(options = {})
     #   prefix: ""                 # Prefix
     #   separator: ""              # Separating prefix and title
     #
-    # === Webdevelopers
-    #
-    # Please use the render_meta_data() helper instead. There all important meta information gets rendered in one helper.
-    # So you dont have to worry about anything.
-    #
-    def render_page_title(options = {})
```

Comment:
Please don't remove the helper now, instead add a depreciation notice and then render the partial.